packages/mc-scripts/src/build.js

```diff
@@ -118,9 +118,16 @@ function build(previousFileSizes) {
           warnings: [],
         });
       } else {
-        messages = formatWebpackMessages(
-          stats.toJson({ all: false, warnings: true, errors: true })
-        );
+        const rawMessages = stats.toJson({
+          all: false,
+          warnings: true,
+          errors: true,
+          moduleTrace: false,
+        });
+        messages = formatWebpackMessages({
+          errors: rawMessages.errors.map((e) => e.message),
+          warnings: rawMessages.warnings.map((e) => e.message),
+        });
```

Comment:
Cool approach. Thinking about it "out load":

Do you think it would be possible to "sniff" the format. So see if we need to map the errors? Should we actually do that? Wondering what happens when `react-dev-utils` gets fixed 🤔.

Reply:
We probably could with some inspection on fields of the `stats.toJson` result. The only thing i think is that when `react-dev-utils` does move to webpack 5, it's likely going to be part of a very large update.

Another proposal, since it's one tiny function, could be to pull `formatWebpackMessages` into this repo and support webpack 5 out of the box?

Reply:
Interestingly, they have a PR open to fix it on their side with the detection, maybe we can lean on that instead:
https://github.com/facebook/create-react-app/pull/10098/files

Reply:
Interesting. Seems like it would continue to work. Maybe you can add a `//TODO: Remove after `react-dev-utils` natively supports webpack@5` to your code so we don't forget :)

Reply:
Done 😄